### DISCUSSION TITLE
A chunkserver process with data has hung up for a long time. Can it be started directly?

### DISCUSSION THREAD

**Discussion by risssse** [moosefs/moosefs#499] [Q&A]
A chunkserver process with data has hung up for a long time. Can it be started directly? This chunkserver has a lot of old data. Will it cause data loss? Will the data of cluster type be overwritten?

**Comment by chogata** [comment]
You can always turn on an old chunkserver, as long as it comes from the same cluster. BUT if your cluster doesn't have undergoals, all the chunks on this server will be either in bad version (because other copies were modified in the meantime) or redundant. So it's probably better to wipe the disks than to connect it "as is", unless i/o operations changing data (aka writes) are minimal on your system.

IMPORTANT: if, at any point, you clicked the "remove" button in CGI for this chunkserver, DO NOT connect this chunkserver with chunks - wipe it and connect it as a new, clean one.
